src/gui/qgsscalewidget.h

```diff
@@ -32,6 +32,8 @@ class GUI_EXPORT QgsScaleWidget : public QWidget
 {
     Q_OBJECT
     Q_PROPERTY( bool showCurrentScaleButton READ showCurrentScaleButton WRITE setShowCurrentScaleButton )
+    Q_PROPERTY( bool scale READ scale WRITE setScale )
```

Comment:
notify is never required, it is just a nice to have.
As far as I know it's not related to MEMBER, do you have a reference for this?